google-cloud-logging/test/google/cloud/logging/project/list_entries_test.rb

```diff
@@ -224,7 +224,7 @@
     list_res = OpenStruct.new(page: OpenStruct.new(response: Google::Logging::V2::ListLogEntriesResponse.decode_json(list_entries_json(3, "next_page_token"))))
 
     mock = Minitest::Mock.new
-    mock.expect :list_log_entries, list_res, [["project1"], resource_names: nil, filter: nil, order_by: nil, page_size: nil, options: default_options]
+    mock.expect :list_log_entries, list_res, [[], project_ids: ["project1"], filter: nil, order_by: nil, page_size: nil, options: default_options]
```

Comment:
In my opinion, we should follow the advice of the API owner and no longer use the deprecated `project_ids` field. In fact, I suspect the best choice at this point (pre-beta) would be a breaking change to our own method signature, removing `project_ids` there as well.